galasa-parent/dev.galasa.framework.api.ras/src/test/java/dev/galasa/framework/api/ras/internal/TestRunQuery.java

```diff
@@ -24,9 +25,48 @@
 
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+import com.google.gson.*;
+
 public class TestRunQuery extends BaseServletTest {	
+
+	private Map<String, String[]> addQueryParameter ( Map<String, String[]> map, String key, String value){
+		if (value != null){
+			map.put(key, new String[] {value});
+		}
+		return map;
+	}
+
+	private Map<String, String[]> addQueryParameter ( Map<String, String[]> map, String key, Integer value){
+		if (value != null){
+			map.put(key, new String[] {value.toString()});
+		}
+		return map;
+	}
+
+	private Map<String, String[]> addQueryTimeParameter ( Map<String, String[]> map, String key, Integer value){
+		if (value != null){
+			String TimeStr = Instant.now().minus(value, ChronoUnit.HOURS).toString();
+			map.put(key, new String[] {TimeStr});
+		}
+		return map;
+	}
+
+	private Map<String, String[]> setQueryParameter (Integer page, Integer size,String sort, String runname, String requestor, Integer agemin, Integer agemax){
```

Comment:
the primitive int does not support a null value, which means that all tests will have age 0 instead of null which will not result in the expected behaviour